pantab/_compat.py

```diff
@@ -1,8 +1,8 @@
-from distutils.version import LooseVersion
+from packaging.version import parse
```

Comment:
It looks like pandas has its own vendored library in `pandas.util.version` - can you use that instead? Would be good to not add a runtime dependency to another library just for this

Reply:
Done.